project/frontend/src/components/PreferenceForm.js

```diff
@@ -105,7 +111,7 @@ function PreferenceForm(props) {
 
   const getSlider = (value, setValue, disabled) => {
     return (
-      <div className='preference-form-slider-container'>
+      <div className='preference-form-slider-container' style={{ paddingLeft: 15}}>
```

Comment:
Could we move the `paddingLeft` style here to the `preference-form-slider-container` class?